pkg/types/room.go

```diff
@@ -97,7 +97,10 @@ This only adds users, and can't remove users from peers.
 */
 func (r *Room) syncPeerLists() {
 	for _, peer := range r.Peers {
-		r.SendMessageToAllPeers(MessageTypeCmd, []byte("join "+peer.RIdentity.Fingerprint()), MessageContentInfo{})
+		r.SendMessageToAllPeers(MessageContent{
+			Type: MessageTypeCmd,
+			Data: []byte("join "+peer.RIdentity.Fingerprint()),
```

Comment:
Use the RoomCommand enum instead of join

Reply:
done